### PR TITLE
Feature/palindrome numbers

### PR DESCRIPTION
**Pull request by stepantzov**

### CODE REVIEW

File: src/training/calculator/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
+package training.calculator;
```
Comment (NZelskaya) [inline src/training/calculator/Calculator.java]: Why it's also here?

File: src/training/numbers/palindrome/NumbersPalindromeLogic.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            reverse = (reverse * 10) + digit;
+            number = number / 10;
+        }
+        while (number != 0);
+
+        if (n == reverse) {
+            flag = true;
+            return flag;
+        }
+        return flag;
```
Comment (NZelskaya) [inline src/training/numbers/palindrome/NumbersPalindromeLogic.java]: it's 5 lines, not 1 :)

File: src/training/numbers/palindrome/NumbersPalindromeLogic.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package training.numbers.palindrome;
+
+/**
+ * Created by Ivan_Stepantsov on 7/16/2017.
+ */
+class NumbersPalindromeLogic {
+    boolean getPalindromeFlag(int number) {
```
Comment (NZelskaya) [inline src/training/numbers/palindrome/NumbersPalindromeLogic.java]: For future - isPolindrome would be better as a name

File: src/training/numbers/palindrome/NumbersPalindromeLogic.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        n = number;
+
+        do {
+            digit = number % 10;
+            reverse = (reverse * 10) + digit;
+            number = number / 10;
+        }
+        while (number != 0);
+        return (n == reverse);
+    }
```
Comment (NZelskaya) [inline src/training/numbers/palindrome/NumbersPalindromeLogic.java]: Finally :)

Comment (stepantzov) [inline src/training/numbers/palindrome/NumbersPalindromeLogic.java]: i've just read the book)

**Pull request merged**